bin/busted

```diff
@@ -4,7 +4,22 @@ setmetatable(_G, nil)
 
 local pl_path = require("pl.path")
 
-local cert_path = pl_path.abspath("spec/fixtures/kong_spec.crt")
+local pl_file = require("pl.file")
+
+local tools_system = require("kong.tools.system")
+
+local cert_path do
+  local busted_cert_file = pl_path.tmpname()
+  local busted_cert_content = pl_file.read("spec/fixtures/kong_spec.crt")
+
+  local system_cert_path, err = tools_system.get_system_trusted_certs_filepath()
+  if system_cert_path then
+    busted_cert_content = busted_cert_content .. pl_file.read(system_cert_path)
```

Comment:
Does this need to anyhow handle possible missing new line(s)?

Reply:
Interesting point. I'll add some extra handling here

Reply:
Fixed, now it follows the logic of `gen_trusted_certs_combined_file` and add newline between certs